pspdev-docker

```diff
@@ -2,8 +2,12 @@
 
 IMAGE="pspdev-docker"
 
+if [ "$EUID" -ne 0 ] && [ -z "$(id|grep docker)" ];then
```

Comment:
This condition doesn't work on macOS (Docker Desktop), where there are no 'docker' group and no need for sudo.

Reply:
Oh that's a good point. Do you have an idea how I could detect the script is running on a Mac? The old version of the script didn't address this either, though, but it's a good idea to improve it.

Reply:
bash defines the variable OSTYPE which can be tested (https://stackoverflow.com/a/18434831)

Reply:
I now check if it's running on Linux and only use sudo if that's the case. Since docker is Linux native I think this should solve most of the edge cases.